src/imitation/rewards/reward_nets.py

```diff
@@ -142,6 +142,28 @@ def predict(
             assert rew.shape == state.shape[:1]
             return rew
 
+    def predict_processed(
+        self,
+        state: np.ndarray,
+        action: np.ndarray,
+        next_state: np.ndarray,
+        done: np.ndarray,
+    ) -> np.ndarray:
+        """Compute the processed rewards for a batch of transitions without gradients.
+
+        Its default behavior in RewardNet is to return the raw rewards from predict().
+
```

Comment:
```suggestion
Subclasses can override this to normalize or otherwise modify the rewards in
ways that may help RL training or other applications of the reward function.

```